### DISCUSSION TITLE
DateTime field - Open on focus, and add Now option

### DISCUSSION THREAD

**Discussion by daanadriaan** [laravel/nova-issues#4650] [Ideas & Feature Requests] [answered]
The DateTime field is a key part of Nova, but I'm missing essential features.

1. Open window on input focus
2. 'Now' option. Today is already there, but I would like a now button.

**Comment by davidhemphill** [comment, marked as answer]
1. Nova uses the browser's in-built widgets for the widest compatibility and consistent user experience. It's safe to assume some accessibility considerations in place that prevented the browser vendors from making the widget from showing on focus.
2. Since we're using the browser widgets, they don't come with this by default, but it could potentially be added to the field components themselves as a button outside.
